addons/pause/addon.json

```diff
@@ -24,6 +24,14 @@
       "matches": ["projects", "projectEmbeds"]
     }
   ],
+  "settings": [
+    {
+      "type": "boolean",
+      "id": "auto-pause",
+      "name": "Pause when page loses focus",
```

Comment:
This setting named like this may be a little unclear to the average user, but I can't think of anything better right now.

Reply:
> This setting named like this may be a little unclear to the average user, but I can't think of anything better right now.

"Pause project when browser tab changes [from Scratch]?"

Reply:
> > This setting named like this may be a little unclear to the average user, but I can't think of anything better right now.
> 
> "Pause project when browser tab changes [from Scratch]"?

What if the user focuses a different _window_?

Reply:
We could potentially go with "pause when project page is minimized".

Reply:
> > > This setting named like this may be a little unclear to the average user, but I can't think of anything better right now.
> > 
> > "Pause project when browser tab changes [from Scratch]"?
> 
> What if the user focuses a different _window_?

Since we're avoiding the word 'focus,' I'm not sure there's much we can do about that. I think this gets the point across well enough, but I don't particularly care.

Reply:
"Pause when leaving tab"?

Reply:
> "Pause when leaving tab"?

Concise, but then users might think that means closing the tab or clicking a link to a different page (which if that were what it did, wouldn't be very helpful).

Reply:
@iqnite Both things can be dealt with in parallel.

Reply:
What about "Auto-pause when switching windows"?

Reply:
> What about "Auto-pause when switching windows"?

But then there's switching tabs.

Reply:
> But then there's switching tabs.

It's technically like switching tabs, and I know browsers that don't even support tabs. Also, many sites talk about "windows" meaning tabs and even Griffpatch says "Close all browser windows and retry".

Reply:
> > But then there's switching tabs.
> 
> It's technically like switching tabs, and I know browsers that don't even support tabs. Also, many sites talk about "windows" meaning tabs and even Griffpatch says "Close all browser windows and retry".

Yes, but all of the browsers we officially support do have tabs, and Griffpatch is referring to windows, not tabs.

Reply:
>@iqnite: and even Griffpatch says "Close all browser windows and retry".

Well thats because closing all windows indirectly closes all tabs

Reply:
Oh boy, I don't think we found a clear way to name this setting 🤦‍♂️

Reply:
> Oh boy, I don't think we found a clear way to name this setting 🤦‍♂️ 

Since the most accurate is "Automatically pause when tab loses focus," I prefer that one.

Reply:
Look at the actual one please.